website/src/index-page.tsx

```diff
@@ -142,15 +142,8 @@ export function IndexPage(): ReactElement {
                     link: '/v3/features/apollo-federation',
                     icon: <SiApollographql size={36} />,
                     title: 'Apollo Federation',
-                    description: (
-                      <>
-                        The{' '}
-                        <TextLink href="/v3/comparison#compatibility-with-apollo-federation">
-                          best supergraph and subgraph
-                        </TextLink>{' '}
-                        for your GraphQL
-                      </>
-                    ),
+                    description:
+                      'The best supergraph and subgraph for your GraphQL',
```

Comment:
Uri wanted this, lets maybe point the title to `/v3/features/apollo-federation` leave the "best supergraph and subgraph" to `/v3/comparison#compatibility-with-apollo-federation`?

Reply:
otherwise, let's add new `onClick` property for `description` block, I think it will fix the hydration error, also inside `onClick` handler you should have a check which html element was clicked

Reply:
This seems to work well https://github.com/dotansimha/graphql-yoga/pull/1940/commits/62d39ab5da21ddef1fafcb568d18a6cc1b178133.